Resources/Prototypes/Entities/Objects/Tools/squeezebottle.yml

```diff
@@ -0,0 +1,129 @@
+- type: entity
+  parent: DrinkBase
+  id: SqueezeBottle
+  name: squeeze bottle
+  description:
+  components:
+  - type: ApplyReagentToItem
+    amountConsumedOnUse: 5
+    onSqueezeNoise: /Audio/Items/squeezebottle.ogg
+  - type: Openable
+    closeable: true
+    sound:
+      collection: packetOpenSounds
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/squeeze_bottle.rsi
+    layers:
+      - state: body
+      - state: cap_closed_red
+        map: ["enum.OpenableVisuals.Layer"]
+      - state: fill1
+        map: [ "enum.SolutionContainerLayers.Fill" ]
+        visible: false
+      - state: front
+        map: [ "enum.SolutionContainerLayers.Overlay" ]
+  - type: Appearance
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+  - type: SolutionContainerVisuals
+    maxFillLevels: 6
+    fillBaseName: fill
+  - type: GenericVisualizer
+    visuals:
+      enum.OpenableVisuals.Opened:
+        enum.OpenableVisuals.Layer:
+          True: {state: "cap_open_red"}
+          False: {state: "cap_closed_red"}
+  - type: TrashOnSolutionEmpty
+    solution: drink
+
+- type: entity
+  parent: SqueezeBottle
+  id: SqueezeBottleOfGlue
+  name: bottle of space glue
+  description: High performance glue intended for maintenance of extremely complex mechanical equipment. DON'T DRINK!
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+          - ReagentId: SpaceGlue
+            Quantity: 30
```

Comment:
Indent consistency. Also check the rest of this file. I'm not going to mark all of them.